Assignment6.txt

```diff
@@ -0,0 +1,3 @@
+1. The code will output 201	1	23.
```

Comment:
This is what would output if the code had been correctly written, yes.  As it is, you'll notice if you try to run it, it'll give you errors.  But otherwise this is correct.